_sass/minimal-mistakes/_variables.scss

```diff
@@ -137,7 +137,7 @@ $medium-wide: 900px !default;
 $large: 1024px !default;
 $large-wide: 1160px !default; 
 $x-large: 1280px !default;
-$max-width: 1400px !default;
+$max-width: $x-large !default;
```

Comment:
@MichaelJCaruso this way you mean I guess.

Reply:
Yes